packages/typespec-ts/src/transform/transfromRLCOptions.ts

```diff
@@ -239,6 +239,7 @@ function extractRLCOptions(
   const includeShortcuts = getIncludeShortcuts(emitterOptions);
   const packageDetails = getPackageDetails(program, emitterOptions);
   const flavor = getFlavor(emitterOptions, packageDetails);
+  emitterOptions.flavor = flavor;
```

Comment:
i prefer not changing this, let's not directly update emitterOptions. as we talked, use the getFlavor for the help.